src_cam/physpkg.F90

```
@@ -1924,6 +2014,15 @@ subroutine tphysac (ztodt,   cam_in,  &
        prec_str(:ncol) = prec_pcw(:ncol) + prec_sed(:ncol)
        snow_str(:ncol) = snow_pcw(:ncol) + snow_sed(:ncol)
 
+       if (compute_enthalpy_flux) then
+          qrain_mg_idx = pbuf_get_index('qrain_mg'     , errcode=i)
+          qsnow_mg_idx = pbuf_get_index('qsnow_mg'     , errcode=i)
```

These should also be moved to `phys_init`. Also, I think the `errcode=i` optional input should be dropped since the return code is not being checked. Without that, a failure results in a call to `endrun` while this version would fail silently.